jac-uk/digital-platform
# #1620 - character issue google doc report

**Author checklist**


---
## What's included?
- Added a callable function to export character issues to a document in Google Drive.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Open Postman and import the latest version of the collection (which can be found in the root of this project)
- Make sure your Postman environment variables contain the correct username and password etc
- Execute the `login` endpoint
- Execute the `exportApplicationCharacterIssues` with a suitable data payload. For example:

```json
{
    "data": {
        "exerciseId": "aRrY8ad5TwiznZZhBqee",
        "stage": "all",
        "status": "all",
        "format": "googledoc"
    }
}
```

- The endpoint should return the path of the file that is created
- Check that a document has been created in the specified path
- Check the content of the document

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Example layout:
https://docs.google.com/document/d/12cTMJEC9j3LbR34j_4cb4iT-oHX-VthQ/edit#

Google drive:
https://drive.google.com/drive/folders/1EgVJQa5zERvpFkAWLa6jgib_8s0YqL1N

## Related permissions
Have permissions been considered for this functionality?
- No permission changes required

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
